Src/Plugins/Security/Security.csproj

```diff
@@ -34,6 +34,7 @@
     <PackageReference Include="GoogleApi" Version="3.10.5" />
     <PackageReference Include="MySqlConnector" Version="1.2.1" />
     <PackageReference Include="Npgsql" Version="5.0.3" />
+    <PackageReference Include="RestSharp" Version="106.11.7" />
```

Comment:
><PackageReference Include="RestSharp" Version="106.11.7" /> [](start = 4, length = 59)

Can't we replace restsharp with  httpclient?

Reply:
I did! We just need to zap this reference, it isn't used. Sorry!

---
In reply to: [575900109](https://github.com/microsoft/sarif-pattern-matcher/pull/210#discussion_r575900109) [](ancestors = 575900109)